src/ncar-bufr2nc-fortran/define_mod.f90

```diff
@@ -343,7 +343,7 @@ subroutine set_name_sensor(instid, sensor)
       case (297); sensor = 'ahi'
       case (207); sensor = 'seviri'
       case (420); sensor = 'airs'
-      case (620); sensor = 'cris'
+      case (620); sensor = 'cris-fsr'
```

Comment:
this seems to be not flexible for `cris` and `cris-fsr` which changed a specific crossover date

Reply:
Yes. @BenjaminRuston I commented below, that I was able to convert a CrIS file from 2020-01-01 using this converter. But that file seems to contain both the old CrIS and CrIS-fsr. For our skylab purposes, I don't think files older than that will need to be converted. Do you agree or have concerns with that?

Reply:
again this converter will hopefully deprecate and we'll move to the EMC one so yes agree for now let's move forward and cross idea of handling older CrIS data when we get our re-analysis funding

Reply:
these numbers for obserrors are `431` long which is `fsr` we would need another vector for the `cris` data for truncated anyway.. 

in short this is hardcoded to `fsr` so this should be fine